README.md

```diff
@@ -1,131 +1,130 @@
+            "format": "iss_sub",
+            "iss": "https://issuer.digitalIdentity.gov/",
+            "sub": "aPairwiseId"
+            }
+         }
+      }
+   }
+}
+```
 
----
+The Security Event Token format extends the JSON Web Token (JWT) format which describes claims. The claims in a Security Event Token are described in [RFC8417](https://datatracker.ietf.org/doc/html/rfc8417) and describe the security event that has taken place, the issuer, the subject and the intended audience of the event.
 
-Setup your environment using the following tools:
+A JWT can be represented as:
 
-- VS Code - [Install VS Code](https://code.visualstudio.com/download)
-- SAM CLI - [Install the SAM CLI](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html)
-- Node.js - [Install Node.js](https://nodejs.org/en/), including `npm`.
-- Docker - [Install Docker community edition](https://docs.docker.com/desktop/)
-- pre-commit - [Install pre-commit](https://pre-commit.com)
-- run `pre-commit install` to ensure the hooks are installed
-  A successful `pre-commit install` will yield the following output:
+- a JSON Web Signature (JWS) where the claims are provided in the payload to be signed (the most common format).
+  - This has the format: `[header].[payload].[signature]`
+- a JSON Web Encryption (JWE) where the claims are provided in the plaintext to be signed and/or encrypted.
+  - This has the format: `[header].[encrypted key].[encrypted payload].[random number].[authentication tag]`
 
-  ```
-  pre-commit installed at .git/hooks/pre-push
-  pre-commit installed at .git/hooks/pre-commit
-  ```
+## Transmitter function
 
-The following extensions should be installed for VSCode:
+The Transmitter function signs and encrypts a Security Event Token and forwards it to the Shared Signals Framework (SSF) Transmitter endpoint. It is triggered when a message is received from the Generator function.
 
-- [AWS Toolkit](https://docs.aws.amazon.com/toolkit-for-vscode/latest/userguide/welcome.html) - The AWS Toolkit can be connected through SSO using `aws configure sso` and then selecting the correct profile at the bottom of VSCode.
+This function:
 
-- [SonarLint](https://marketplace.visualstudio.com/items?itemName=SonarSource.sonarlint-vscode) - SonarLint should be connected through the `Connected Mode` it offers. Once it has been connected, creating the binding of your local repo to the `di-fraud-mock-rp` repo on `SonarCloud`.
-  This should add `problems`, which is accessed by `View` -> `Problems`.
+1. Signs the Security Event Token, using a private key as the signature. The signed SET and signature are used to generate a JWS object. The SSF endpoint will use the corresponding public key to verify the signature.
+2. Encrypts the JWS, using the full JWS object as the message:
+   1. Generates a random Content Encryption Key (CEK).
+   2. Encrypt the CEK with the RP's public key to form the JWE encrypted key.
```

Comment:
The payload is signed and not encrypted?

Reply:
I've spoken with Dan Lewis to get more clarification around what the transmitter function is doing. He confirmed that we used to do encryption but no longer do and also gave information about how cognito is used to generate an auth key. I've updated the docs to reflect this.